<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QMainWindow" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="objCategoria">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Seleccionar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Basico</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aseo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>General</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Inventario Tienda My Barrio</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Codigo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Costo Unitario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Categoria</string>
    </property>
   </widget>
   <widget class="QPushButton" name="objIngresar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="objListar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="objSalida">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>771</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="objCodigo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="objCantidad">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="objNombre">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="objCosto">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
